app/scripts/modules/core/src/storage/storageAccount.service.ts

```diff
@@ -0,0 +1,15 @@
+import {module} from 'angular';
```

Comment:
Space around the imports here and below.

Reply:
Done.